### ISSUE TITLE
[Feature Request] When having multiple studies have a different color outline on viewports

### ISSUE THREAD

**Issue by NurGVaras** [OHIF/Viewers#4031]
### What feature or change would you like to see made?

We are using the functionality to show multiple studies at the view giving different StudyInstancesUIds, the problem is that the professionals use the viewer and put the images in the same layout, and they want to distinguish between studies to minimize errors.

Any chance that in the viewports use different colored outlines for different StudyInstancesUIds? 

### Why should we prioritize this feature?

This would be helpful to reduce cross examination errors

**Comment by sedghi** [comment]
I think it is useful feature, and i have seen it in other viewers too. Although i have seen they have different color for overlay text and not bprder

**Comment by NurGVaras** [comment]
yeah that could be useful too, the main issue is to show that are from different studies to avoid cross examinations

**Comment by NurGVaras** [comment]
yeah the idea is to show that are from different studies to avoid cross examinations, that could work too, sorry for the miss click

**Comment by sedghi** [comment]
@salimkanoun Could you help us gather requirements for this project? What insights have you observed from viewers?

**Comment by salimkanoun** [comment]
We did this in our distribution, it can be donne by adding a custom attribute to the Hanging protocol for instance "borderColor" and make a custom viewport that will inject this property in the rendered viewport.

So basically in OHIF you can just add a default "borderColor" porperty to the defautl viewport and add the tailwind classname according to the predefined list of colors.

Then each user can use or not this attribute to make this color border and chose it's own color.

I don't think viewport should have this by default, colorborder can also overload display not everyone will like it.

Other way could be to change color of the seriesDescription, or seriesDate to differentiate viewport.

The point is, OHIF is flexible enough to do all of this by customizing viewport and using custom HP properties.

Eventually add an optional borderColor properties can make it easier to implement
